sections/plate_notation.tex

```diff
@@ -0,0 +1,61 @@
+Unless otherwise stated, future mentions of the author-topic model are to our implementation of this model, rather than the model presented by \citet{author_topic_2012}.
+The plate notation for our category and author \gls{lda} models can be seen in \autoref{fig:metadata_lda}.
+
+\subsection{Pachinko Allocation}\vejleder{make sure that you talk about the original pachinko model and not our modification}
+In order to handle the hierarchical structure of the taxonomy metadata field, we use a hierarchical topic model, namely the \acrfull{pam} from \citet{li2006pachinko}.
+Pachinko allocation generalizes \gls{lda}, making it possible to construct topic hierarchies based on any \gls{dag} structure.
+\gls{pam} is a topic model focusing on finding topics of different abstraction levels and modeling the connections between these topics.
+
+Each node in the \gls{dag} structure represents a topic in the pachinko allocation model. 
+However, unlike \gls{lda} where topics are distributions over words, in \gls{pam} topics are multinomial distributions over words and/or other topics further down the hierarchy of the \gls{dag} structure.
+An example of the \gls{dag} structure used in this paper is in \autoref{fig:pachinko_dag} where each level of the \gls{pam} is shown.
+The idea behind the \gls{dag} structure is to be able to model correlations between topics and in turn make more coherent topics.
+  
+In this paper, we use a layered \gls{pam}, as in \cite{li2006pachinko}, meaning that the \gls{dag} structure is divided into layers where each layer is fully connected.
+However, \citet{li2006pachinko} use four layers where we use five to capture more taxonomy fields.
+
+We replicate some layers in our \gls{dag} structure based on the structure from the taxonomy field within our dataset, having some nodes represent a topic based on a specific taxonomy.
+An example of this can be seen in \autoref{fig:pachinko_dag}, where we have the node "STEDER", and this is connected to "Danmark" in the third layer.
+To make the algorithm construct the topics to be based on our taxonomies, we introduce a novel locking mechanism for the Gibbs sampler which we use to run \gls{pam}.
+This mechanism is discussed further in \autoref{subsec:mod_pachinko}.
+
+We use a five-level\vejleder{hvorfor bruger vi en 5 lag} pachinko tree structure, following the format presented by \citet{li2006pachinko}.
+The first layer is the root layer which all topics are a part of.
+The last layer is the word layer consisting of one node for each word in the vocabulary of our corpus.
+The second and third layers will be constructed based on the entries of the first two positions in our taxonomy metadata, meaning there will be one node for each unique sub-taxonomy entry that is in the first or second position in the taxonomy sequence (e.g. "STEDER" and "Danmark", which is taken from "STEDER/Danmark/Aalborg", but not "Aalborg" since it is in the third position).
```

Comment:
```suggestion
The second and third layers will be constructed based on the entries of the first two positions in our taxonomy metadata, meaning there will be one node for each unique sub-taxonomy entry that is in the first or second position in the taxonomy sequence (e.g., "STEDER" and "Danmark", which is taken from "STEDER/Danmark/Aalborg", but not "Aalborg" since it is in the third position).
```